docs/man/docker-build.1.md

```diff
@@ -36,6 +38,19 @@ as context.
 **-f**, **--file**=*Dockerfile*
    Path to the Dockerfile to use. If the path is a relative path then it must be relative to the current directory. The file must be within the build context. The default is *Dockerfile*.
 
+**-e**, **--env**=*environment*
+   Set build-time environment variables. This option allows you to specify
+arbitrary environment variables that are available for the command(s) that will
+be executed as part of 'RUN' primitive of Dockerfile. Such a variable is only
```

Comment:
I think this would need to change too based on how we want the env vars to be available for more than just RUN.